apps/design/frontend/src/ballot_viewer.tsx

```diff
@@ -384,42 +485,84 @@ export function BallotViewer({
               <div>
                 {grid.columns} columns x {grid.rows} rows
               </div>
-              <div style={{ marginTop: '0.25rem' }}>
-                <label style={{ cursor: 'pointer' }}>
-                  <input
-                    type="checkbox"
-                    checked={showGridLines}
-                    onChange={() => setShowGridLines(!showGridLines)}
-                  />{' '}
-                  Show grid lines
-                </label>
-              </div>
             </div>
           </Column>
-          <P />
         </section>
-        <div style={{ flexGrow: 1 }} />
-        <section>
+        <section style={{ marginTop: 'auto' }}>
           <H3>Export</H3>
-          <P>
-            <Button
-              color="inverseNeutral"
-              onPress={onExportPress}
-              disabled={exportBallotMutation.isLoading}
-            >
-              Export PDF
-            </Button>
-          </P>
+          <Button
+            color="inverseNeutral"
+            onPress={onExportPress}
+            disabled={exportBallotMutation.isLoading}
+          >
+            Export PDF
+          </Button>
         </section>
       </Controls>
       <Canvas ref={measureRef}>
         {dimensions && (
           <DocumentSvg
             dimensions={dimensions}
             document={ballotResult.ok().document}
-            grid={showGridLines ? grid : undefined}
+            debugInfo={{
+              ...debugFlags,
+              gridLayout: ballotResult.ok().gridLayout,
+              grid,
+            }}
           />
         )}
+        <DebugPanelWrapper>
```

Comment:
Consider extracting out of `BallotViewer` even if it stays in this file?

Reply:
01cc133ba